### ISSUE TITLE
No config applied in version 2.0.2

### ISSUE THREAD

**Issue by LorisZ** [dpa99c/cordova-custom-config#64]
We use this plugin to change parameters for AndroidManifest.xml and it worked just fine in 2.0.0. But using the version 2.0.2 it stopped working; the config was missing completely in the AndroidManifest.xml found inside platforms.

short excerpt very similar to our config we use:
```xml
<platform name="android">
        ...
        <config-file target="AndroidManifest.xml" parent="/*">
            <uses-permission android:name="android.permission.CAMERA" />
            <uses-feature android:name="android.hardware.camera" />
            ...
        </config-file>
        <config-file target="AndroidManifest.xml" parent="./application">
            <activity android:label="@string/title_activity_photo_gps" android:name="com.example.PhotoGpsActivity" android:screenOrientation="portrait" android:windowActionBar="false"/>
            <activity android:name="com.example.NfcActivity" android:theme="@android:style/Theme.Black.NoTitleBar" />
        </config-file>
    </platform>
```

Can someone reproduce this bug? Or was there some change to the syntax/behavior? I wouldn't expect such thing to happen on a patch level change.

**Comment by dpa99c** [comment]
Using your example config above, I can reproduce an issue where the Cordova activity is being overwritten with custom activities. I have tracked this down to a bug introduced by pull request #48. I'll fix this and push out a patch release as 2.0.3

**Comment by dpa99c** [comment]
Please check that cordova-custom-config@2.0.3 resolves your issue.
